index.js

```diff
@@ -9,11 +30,15 @@ const head = {
     { view: "label", label: "My App", css: "logo" },
     {
       view: "button",
+      id: "profile-button",
       label: "Profile",
       type: "icon",
       icon: "wxi-user",
       width: 150,
       css: "profile-icon",
+      on: {
+        "onItemClick": showPopup
```

Comment:
This **does not** require any changes (solution is valid), but FYI has an alternative: 
https://docs.webix.com/desktop__popup.html#creatingpopupmenu
https://snippet.webix.com/251f4e5b